packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -1223,6 +1223,63 @@ void main() {
 
     expect(item0, findsNothing);
   });
+
+  testWidgets('SliverReorderableList auto scrolls speed is configurable', (WidgetTester tester) async {
+    Future<void> pumpFor({ required Duration duration, Duration interval = const Duration(milliseconds: 50), }) async {
```

Comment:
Since the auto scroll happens dispatching `animateTo` one after the other, I could not find any other way to let them execute apart from pumping with some frequency.
- pumpAndSettle does not work because it never settles, it would only settle when the list got to the end, which I don't wan't.
- pump with a big value does not work because it won't let all the `animateTo` to happen, but only 1 of them.

Please let me know if there is a better way to achieve this.